test/helpers/BytesHelper.js

```diff
@@ -24,7 +42,27 @@ function zeroExOrderToBytes(order) {
   return web3Instance.utils.bytesToHex(v);
 }
 
+function zeroExV2OrderToBytes(order) {
+  const v = []
+    .concat(toBytes32(order.maker))
```

Comment:
All of these variable names need to be consistent with https://0xproject.com/docs/0xjs#types-SignedOrder here and all throughout the 0x v2 tests